pkg/reconciler/inmemorychannel/controller/resources/dispatcher.go

```diff
@@ -121,5 +126,11 @@ func makeEnv() []corev1.EnvVar {
 				FieldPath: "metadata.namespace",
 			},
 		},
+	}, {
+		Name:  "MAX_IDLE_CONNS",
+		Value: strconv.Itoa(dispatcherConfig.MaxIdleConns),
```

Comment:
Where do you check the string can be converted to a int?

Reply:
It's done here i guess https://github.com/knative/eventing/pull/4543/files#diff-9b49373f252294fb9fd09a0678d4c39a6cb5444968761b0ff12d6145fd1d96fa

Reply:
yup, thanks!